api/libretimeapi/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for api project.
```

Comment:
It doesn't. ASGI is probably a better solution long term, but I don't know hoq to set it up

Reply:
Similar to wsgi you need a server. There seem to be [three asgi servers](https://docs.djangoproject.com/en/3.1/howto/deployment/asgi/) rn:
* daphne
* hypercorn
* uvicorn
I did a quick check on pkgs.org and daphne and uvicorn are both available on various distros with uvicorn being around since buster on Debian. For what [I gathered](https://dev.to/bowmanjd/the-three-python-asgi-servers-5447) uvicorn is the only one out of the bunch that does not support HTTP/2 but all of them support websockets and HTTP/1.

IMO it looks like switching from the WSGI protocol to ASGI isn't something we need to solve right now as there is no pressing need for HTTP/2 in our use-case.